README.md

```diff
@@ -11,7 +11,7 @@ cowsay
 If you are familiar with `cowsay` on the cli, then you know what this
 is, but for R. If not, read below. Why? Why not?
 
-### Contributors (alphebetical)
```

Comment:
@mnr please undo this change - although it needs to be fixed, PR's should stay focused on one issue instead of pulling in other fixes

Reply:
agreed and done. Revised pull to come soon...